### DISCUSSION TITLE
Nothing happens when running the installer.

### DISCUSSION THREAD

**Discussion by orrd** [valinet/ExplorerPatcher#531] [General]
Nothing happens when I run the installer (downloaded from https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe).  

I have a fresh install of Windows 11 with just a few apps installed (including Chrome and Dell's SupportAssist).  I also tried running it as Administrator.  It just doesn't do anything, no prompts, etc. 

Am I missing something obvious?

**Comment by BraINstinct0** [comment]
Maybe...something like restarting?
IDK what your status is at but first try checking if ExplorerPatcher shows up in the 'installed programs' in Settings. If so, it is there. If not, it isn't.
Also, the options menu can be accessed by right-clicking the taskbar, then selecting 'Properties'(or, options, or...settings... or...I don't recall what it's called, but it'll be `the third entry counted from the bottom`)

**Comment by orrd** [comment]
Ok, it does seem to be working now.  I'm not sure what I did differently (possibly doing a restart did it).   Thanks for the reply.

**Reply by BraINstinct0** [reply]
Np 😜 Good to know it's working well now.
